lib/api/controllers/routerController.js

```diff
@@ -370,8 +370,26 @@ module.exports = function RouterController (kuzzle) {
     });
 
     // create and mount a new router for plugins
-    this.pluginRouter = new Router();
-    api.use('/_plugin', this.pluginRouter);
+    api.use('/_plugin', apiPlugin);
+
+    kuzzle.pluginsManager.routes.forEach(route => {
+      apiPlugin[route.verb](route.url, function (request, response) {
```

Comment:
the callback function is the same as for non-plugin routes: it should be factorized to avoid code duplication.